app/components/Form/ProjectFundingAgreementForm.tsx

```diff
@@ -141,10 +150,16 @@ const ProjectFundingAgreementForm: React.FC<Props> = (props) => {
     `,
     props.query
   );
-
   const fundingAgreement =
     projectRevision.projectFundingAgreementFormChanges.edges[0]?.node;
 
+  const calculatedProponentsSharePercentage: number = fundingAgreement
```

Comment:
Can we make this into an exported function instead of a conditional? We use it in the summary as well, and it will save having to maintain the logic in more than  one place